src/components/ChildIncidents/index.js

```diff
@@ -102,15 +103,25 @@ const Li = styled(ListItem)`
 
 const ChildIncidents = ({ className, incidents }) => (
   <List className={className} data-testid="childIncidents">
-    {incidents.map(({ href, status, values, changed }) => {
-      const valueEntries = Object.entries(values).map(([key, value]) => (
-        <DisplayValue key={key} title={value}>
-          {value}
-        </DisplayValue>
-      ));
+    {incidents.map(({ href, status, values: childIncident, changed }) => {
```

Comment:
Instead of reassigning `values` to `childIncident`, it would be cleaner to assign the 'correct' key in the ChildIncidents component directly. Besides that, I think that the 'values' data could be moved to the root.

Reply:
Good point Maurits! Gets more readable in this way.

Reply:
To make it less confusing and 'more readable' I un-destructured everything except props.